### PR TITLE
Adding Lab 02 Initial Draft

### PR DESCRIPTION
**Pull request by rechhabra**
Adding my initial draft for lab 02: pandas lab for 7 February 2023, consisting of a README, a jupyter notebook for lab interaction, and the self-made csv dataset.

### CODE REVIEW

File: labs/02-lab/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+07 February 2023
+
+
+### Overview
+In this lab, you will be learning how to use the Pandas library to read in data from a CSV file and perform some basic data analysis and visualization.
+
+### Getting Started
+1. See the `lab02.ipynb` file in the `02-lab` folder. This is the file you will be working in.
+2. Make sure you have the `cookies.csv` file in the same folder as the `lab02.ipynb` file.
+3. Make sure you have the `pandas` library installed. If you don't, you can install them using `pip install pandas` and `pip install matplotlib` in your notebook environment or your terminal. For your convenience, I added the installation commands to the first cell of the notebook.
```
Comment (gallettilance) [inline labs/02-lab/README.md:11]:
```suggestion
3. Make sure you have the `pandas` and `matplotlib` libraries installed. If you don't, you can install them using `pip install pandas` and `pip install matplotlib` in your notebook environment or your terminal. For your convenience, I added the installation commands to the first cell of the notebook.
```

File: labs/02-lab/lab02.ipynb [hunk 2]
```diff
@@ -0,0 +1,579 @@
...
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Overview\n",
+    "In this notebook, you will be learning how to use the Pandas library by working with the `cookies.csv` file. \n",
+    "\n",
+    "#### `cookies.csv` file\n",
+    "\n",
+    "The `cookies.csv` file contains information about cookies that were made from a single Rico's Bisquito's factory. What different from homework0, however, is that there are a couple differences from the proposed classes.\n",
```
Comment (gallettilance) [inline labs/02-lab/lab02.ipynb:41]:
```suggestion
    "The `cookies.csv` file contains information about cookies that were made from a single Rico's Bisquito's factory. There are, however, a few differences from the classes defined in homework0.\n"
```

File: labs/02-lab/lab02.ipynb [hunk 3]
```diff
@@ -0,0 +1,579 @@
...
+    "#TODO: check the data types of the columns"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Cleaning the data\n",
+    "Now that we have the data, we need to clean it. For example, some `cost_to_make` fields of some created cookies are missing. To resolve this, we can do many things: we can remove the rows with missing data, we can replace the missing data with the mean of the column, or we can get rid of the row entirely. "
```
Comment (gallettilance) [inline labs/02-lab/lab02.ipynb:139]:
```suggestion
    "Now that we have the data, we need to clean it. For example, some `cost_to_make` fields of some created cookies are missing. To resolve this, we can do many things: we can replace the missing data with the mean of the column, or we can get rid of the row entirely if the `cost_to_make` field is not set. "
```

File: labs/02-lab/lab02.ipynb [hunk 4]
```diff
@@ -0,0 +1,579 @@
...
+    "\n",
+    "#TODO: use apply() to adjust the cost_to_make column."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And we do a lot more things! We will see these concepts be used in the next homework assignment, along with a couple new ones to show you how powerful Pandas is."
```
Comment (gallettilance) [inline labs/02-lab/lab02.ipynb:286]:
```suggestion
    "And we can do a lot more things! We will see these concepts be used in the next homework assignment, along with a couple new ones to show you how powerful Pandas is."
```

**Review by gallettilance — APPROVED** [review]
Looks great! Just a few typos

File: labs/02-lab/README.md (hunk 1 above)
Comment (rechhabra) [reply labs/02-lab/README.md:11]: Actually, I did not mean to put `matplotlib` in this lab --- I forgot to remove it from the second sentence.

**Pull request merged**